board/kontron/sa67/sa67.env

```diff
@@ -1,3 +1,5 @@
+#include <env/ti/k3_dfu.env>
```

Comment:
This also includes the layout of the eMMC, which doesn't align with ours. Just duplicate that file and drop the all the environment variables except for the dfu_alt_info_ram. (see the phytec k3_dfu.env for example)